CS_Ed_Week/CS_Ed_Week.md

```diff
@@ -8,75 +8,101 @@ A workshop for getting young people interested in computer science.
 - This "true or false" logic is the basis of all of programming and is the core of several algorithms.
   - It's also important to mention that this is the basis of all computers as well.
   - You may have seen it, it's binary.
-  - 1 equates to "true" while 0 equates to false.
+  - 1 equates to "True" while 0 equates to false.
+  - Remember, the "True" or "False" must be capitalized to be counted as a boolean, otherwise it will through an error.
+- Example:
+  - `is_it_true = True`
+  - `is_it_false = False`
+  - `enabled = True`
 
 ### Printing data to stdout
 
-- Many programs rely on some sort of output. That output can take many forms, but one of the most common is just text, also called standard output.
-- You can print any data type with the print() function:
+- Many programs rely on some sort of output. That output can take many forms, but one of the most common is just text, also called standard output (stdout).
+- You can print any type data with the print() function:
   - `print(5)`
   - `print("foo")`
   - `print(True)`
+  - `print([1,2,3])`
 - You can also print the value of a variable by using the variable's name as the first argument to print():
   - `bar = "hello world"`
   - `print(bar)`
-- Sometimes you may want to print a message that relies on the value of a variable. For instance, you have a variable containing a username, and every time the user logs in, you want to display a personalized welcome message.
-- For this, you use format string, also called f-strings:
+- If you wish to use multiple different types in the same line, you would have to use a format string (f-string).
+- These take a variable and put it directly into the string, no matter the type.
+- The syntax is `f" "`
+- Example:
   - `username = "Joe"`
-  - `print(f"Welcome back {username}!")`
+  - `print(f"Welcome back {username}!") # "Welcome back, Joe!"`
+  - `number = 7`
+  - `print(f"{username}, this talk ends at {number}:00") # "Joe, this talk ends at 7:00"`
+
+### Comments
+
+- Python has one main comment type, which is the "hash-comment"
+- Simply put a hash (`#`) sign on a line, and the python interpreter will skip everything after and go to the next line.
+- There's also tripple-quote "comments", but those are more designed for things like docs, so we won't go into them here.
 
 ### Arrays vs. Linked Lists
 
 - Computer memory resembles a grid of slots, each with its own address.
 - When you want to store multiple multiple items in memory, you can can use either arrays or linked lists.
-- Elements in an array are stored one after another in memory. They cannot be divided up and then stored in different locations. If you run out of room for your array, you must find a bigger block of memory.
+- Elements in a list are stored one after another in memory. They cannot be divided up and then stored in different locations. If you run out of room for your array, you must find a bigger block of memory.
```

Comment:
I think the distinction between arrays and linked lists is clearer when we refer to arrays as arrays, and not as lists. I know they are called lists in Python, so maybe we can make a note that arrays are called lists in Python, but in general computer science, I think they are simply called arrays. 

Basically, list is too similar to linked lists\, and both will probably be new concepts, so we don't want to confuse.